ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml

```diff
@@ -6757,7 +6757,7 @@ periodics:
 - agent: kubernetes
   cluster: api.ci
   decorate: true
-  interval: 4h
+  interval: 12h
```

Comment:
is this 4->12 intentional?

Reply:
or is it to compensate for increasing in the other the metal job above?

Reply:
Yeah it was intentional though out of line with PR description and commit message. Basically this was a new job and we were trying to get a lot of runs in quickly but it's been running for a month now so no longer need such rapid feedback.

Reply:
makes sense wfm